artiq/gui/entries.py

```diff
@@ -99,6 +99,9 @@ def state_to_value(state):
     def default_state(procdesc):
         if "default" in procdesc:
             return procdesc["default"]
+        elif "min" in procdesc and procdesc["min"]:
```

Comment:
Why are you also testing for ``procdesc["min"] != 0``?

Reply:
oh, it is actually doing `procdesc["min"] is None`

Reply:
That's not how Python works. It would also test for ``!= 0``.

Reply:
right. i didn't remember it is an Option[int] here, yet they overlapped when they shouldn't